app.ts

```
@@ -49,19 +49,19 @@ app.use(apiRateLimitMiddleware);
+			method: req.method,
 			path: req.path,
-			ipHash: fingerprint.ipHash,
+			statusCode: 404,
+			fingerprint,
+		}))
+		.catch((error) => {
+			logger.error('Failed to persist not-found security event', {
+				error: error instanceof Error ? error.message : String(error),
+				path: req.path,
+			});
 		});
```

ESLint

[error] Expected 'undefined' and instead saw 'void'.

(no-void)